gulpfile.js

```diff
@@ -30,12 +31,18 @@ gulp.task('copySource', gulpTasks.copy({
   src: `${SOURCE_DIR}/guide/**/*.jsx`,
 }).task);
 
-gulp.task('deploy', gulpTasks.deploy({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID || 'No ENV',
-  bucketName: process.env.AWS_BUCKET_UI_FRAMEWORK || 'No ENV',
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || 'No ENV',
-  src: `${DISTRIBUTION_DIR}/**/*.*`,
-}).task);
+gulp.task('deployToGitHubPages', () => (
+  gulp.src('./dist/**/*')
+    .pipe(ghPages())
+));
+
+gulp.task('deploy', done => {
+  runSequence(
+    'production',
+    'deployToGitHubPages',
+    done
+  );
+});
```

Comment:
I think this task reduces the flexibility for deployments (especially if we decide to keep the AWS task as well). What's the benefit?